features/base/media/components/native/Video.js

```diff
@@ -42,13 +43,26 @@ export class Video extends Component {
             let style = styles.video;
             let objectFit = (style && style.objectFit) || 'cover';
 
-            return (
+            let video = (
                 <RTCView
                     objectFit={ objectFit }
                     streamURL={ streamURL }
                     style={ style }
                 />
             );
+
+            // XXX RTCView does not currently support mirroring for local
+            // videos, even when providing a transform style property.
+            // We can achieve the same effect by wrapping the RTCView inside of
+            // another View, and applying the transform style property to that
+            // View instead.
+            if (this.props.mirror) {
+                return (
+                    <View style={ styles.mirroredVideo }>{video}</View>
```

Comment:
We could certainly extract more methods, but it feels more complex than necessary here.